packages/react/menu/src/Menu.tsx

```diff
@@ -521,7 +508,10 @@ const MenuContentImpl = React.forwardRef((props, forwardedRef) => {
                 loop={loop}
                 currentTabStopId={currentItemId}
                 onCurrentTabStopIdChange={setCurrentItemId}
-                onEntryFocus={onEntryFocus}
+                onEntryFocus={(event) => {
```

Comment:
This only takes into account the "focus first item" right? are we going to do the focus last when pressing up?

Reply:
oh bum, hadn't thought of that 🤔  we should do... hmmm

Reply:
That one would have to be handled as an `onKeyDown` on the trigger I guess, but the issue is the items tracked are inside `Menu` right? not inside `DropdownMenu`. We could potentially export the `useCollection` hook from `Menu` to get them in `DropdownMenu`… 🤔

Reply:
We agreed we're going to solve this in a separate PR right? I'll add a bullet to the `submenu` PR description if so

Reply:
yep 👍